### PR TITLE
Discord Worker

### PR DESCRIPTION
**Pull request by kevinz917**
Summary of changes
- Added worker used for Discord verification. This worker uses the public API from Discord to receive the message through a channelId, userId, and also a secret key used by the Discord Bot, which must be present in the verification server. 
- Refactored build_twitter_vc to the corresponding twitter.rs file.

### CODE REVIEW

File: worker/src/lib.rs [hunk 1]
```diff
@@ -75,49 +76,18 @@ fn build_instagram_vc_(pk: &JWK, instagram_handle: &str) -> Result<Credential> {
...
 fn verify_signature(data: &str, pk: &JWK, sig: &str) -> Result<()> {
     info!("Verify tweet signature.");
     let micheline_data = ssi::tzkey::encode_tezos_signed_message(data)?;
     let (algorithm, sig_bytes) = ssi::tzkey::decode_tzsig(sig)?;
     Ok(verify_bytes(algorithm, &micheline_data, pk, &sig_bytes)?)
 }
 
+fn initialize_logging() {
+    use log::Level;
+    console_log::init_with_level(Level::Trace).expect("error initializing log");
```
Comment (sbihel) [inline worker/src/lib.rs:88]: Put it back to `Level::Error`

File: worker/src/lib.rs [hunk 2]
```diff
@@ -269,6 +242,90 @@ pub async fn witness_tweet(
...
+
+        // Check for matching handles
+        if discord_handle != formatted_discord_handle {
+            jserr!(Err(anyhow!(format!(
+                "Different Discord handle {} v. {}",
+                discord_handle, formatted_discord_handle
+            ))));
+        }
+
+        let (sig_target, sig) = jserr!(twitter::extract_signature(discord_res.content));
```
Comment (sbihel) [inline worker/src/lib.rs:277]: That function should probably move out of the twitter module to avoid any confusion

**Review by sbihel — APPROVED** [review]
BTW you didn't need to close the previous PR, it's ok to force push if you made a mistake

File: worker/src/lib.rs [hunk 3]
```diff
@@ -75,49 +76,18 @@ fn build_instagram_vc_(pk: &JWK, instagram_handle: &str) -> Result<Credential> {
     }))?)
 }
 
-fn build_twitter_vc_(pk: &JWK, twitter_handle: &str) -> Result<Credential> {
```
[the remark this answers is not in the document]
Comment (kevinz917) [reply worker/src/lib.rs:78]: Migrated to instagram.rs

File: worker/src/lib.rs (hunk 2 above)
Comment (kevinz917) [reply worker/src/lib.rs:277]: Refactored to a standalone function in lib.rs

**Pull request merged**